packages/menu/src/Menu.ts

```diff
@@ -63,13 +63,11 @@ export class Menu extends SpectrumElement {
     }
 
     public focus(): void {
-        if (this.menuItems.length === 0) {
+        if (!this.menuItems.length) {
```

Comment:
Why did you need this change -- is this just a code style choice, or is there something subtle I'm missing?

Reply:
Just style, I had removed this check at one point and then re-added it not knowing the style changed. There _may_ be some slight amount of type security outside of the TS space in the small fraction of time `this.menuItems === undefined`, but I don't know enough V8/timing to confirm.